Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/program/database/data/PointerDBAdapterV3.java

```diff
@@ -0,0 +1,124 @@
+/* ###
+ * IP: GHIDRA
+ * REVIEWED: YES
```

Comment:
I don't think that is true :laughing:

Reply:
I just copied PointerDBAdapterV2.java, and didn't look at the header :)